webapp/src/components/Vendor/decentraland/NFTFilters/NFTFilters.tsx

```diff
@@ -337,19 +340,32 @@ const NFTFilters = (props: Props) => {
           minWidth={Responsive.onlyTablet.minWidth}
           className="filters"
         >
-          <FiltersMenu
-            assetType={assetType}
-            selectedNetwork={network}
-            selectedCollection={contracts[0]}
-            selectedRarities={rarities}
-            selectedGenders={wearableGenders}
-            isOnlySmart={!!onlySmart}
-            onCollectionsChange={handleCollectionsChange}
-            onGendersChange={handleGendersChange}
-            onRaritiesChange={handleRaritiesChange}
-            onNetworkChange={handleNetworkChange}
-            onOnlySmartChange={handleToggleOnlySmart}
-          />
+          {category === NFTCategory.WEARABLE && (
+            <FiltersMenu
+              assetType={assetType}
+              selectedNetwork={network}
+              selectedCollection={contracts[0]}
+              selectedRarities={rarities}
+              selectedGenders={wearableGenders}
```

Comment:
same as the other comment
```suggestion
              selectedGenders={isWearable ? wearableGenders : undefined}
```